integration/src/test/scala/org/knora/webapi/responders/v2/ResourcesResponderV2Spec.scala

```diff
@@ -2311,29 +2305,14 @@ class ResourcesResponderV2Spec extends CoreSpec with ImplicitSender {
       ).map(_.toSmartIri)
 
       for (erasedIriToCheck <- erasedIrisToCheck) {
-        val sparqlQuery = org.knora.webapi.messages.twirl.queries.sparql.admin.txt
-          .checkIriExists(
-            iri = erasedIriToCheck.toString
-          )
-          .toString()
-
-        appActor ! SparqlAskRequest(sparqlQuery)
-
-        expectMsgPF(timeout) { case entityExistsResponse: SparqlAskResponse =>
-          entityExistsResponse.result should be(false)
-        }
+        val query = sparql.admin.txt.checkIriExists(erasedIriToCheck.toString)
+        UnsafeZioRun.runOrThrow(TriplestoreService.query(Ask(query))) should be(false)
       }
 
       // Check that the deleted link value that pointed to the resource has also been erased.
-      val isEntityUsedQuery = org.knora.webapi.messages.twirl.queries.sparql.v2.txt
-        .isEntityUsed(
-          entityIri = resourceIriToErase.get.toSmartIri.toInternalIri,
-          ignoreKnoraConstraints = true
-        )
-
-      appActor ! SparqlAskRequest(isEntityUsedQuery)
-
-      expectMsgPF(timeout) { case SparqlAskResponse(isUsed) => assert(!isUsed, s"Link value was not erased") }
+      val query =
+        sparql.v2.txt.isEntityUsed(resourceIriToErase.get.toSmartIri.toInternalIri, ignoreKnoraConstraints = true)
+      UnsafeZioRun.runOrThrow(TriplestoreService.query(Ask(query))) should be(false)
```

Comment:
Did the response type change? 
before: `assert(!isUsed, s"Link value was not erased")`
now: `false`

Reply:
`assert(someBoolean, ...)` expects `someBoolean` to be `true`. `!someBoolean` would be `false`, wouldn't it?

Reply:
Correct, I was more about message `s"Link value was not erased"` now isn'r checked.